### PR TITLE
Super Quickstart Notes

### PR DESCRIPTION
**Pull request by cs-499243**
Explains things so Matt doesn't have to!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ MyRadio is part of a suite of upcoming public projects, including:
 - loggerng, our python audio logging and retriving system
 - Bootstrapping scripts for setting up and configuring all the dependencies
 
-Super Quickstart
+Super Quickstart [@]
```
Comment (markspolakovs) [inline README.md:14]: Why the @'s?

Comment (cs-499243) [reply README.md:14]: Just a reference point - could remove it it's just how I like to refer to things

Comment (michael-grace) [reply README.md:14]: i thought this, but am just 💩

File: README.md [hunk 2]
```diff
@@ -11,9 +11,9 @@ MyRadio is part of a suite of upcoming public projects, including:
...
 ----------------
-MyRadio comes with a Vagrantfile based on Ubuntu 16.04.
+MyRadio comes with a Vagrantfile based on Ubuntu 16.10.
```
Comment (mstratford) [inline README.md:16]: 19.10

File: README.md (hunk 1 above)
Comment (markspolakovs) [reply README.md:14]: bin pls :)

**Comment by mstratford** [comment]
Poke?

**Comment by markspolakovs** [comment]
@cs-499243 could you make the changes above mebbe please?

**Pull request merged**